crates/oxc_transformer/tests/integrations/plugins/replace_global_defines.rs

```diff
@@ -68,7 +67,22 @@ fn dot_with_overlap() {
     ])
     .unwrap();
     test("import.meta.env", "__foo__", config.clone());
-    test("import.meta.env.NODE_ENV", "import.meta.env.NODE_ENV", config.clone());
```

Comment:
Shouldn't we keep this test, but change it's output?

```rs
test("import.meta.env.NODE_ENV", "__foo__.NODE_ENV", config.clone());
```

Reply:
Done